docs/reference/modules/discovery/quorums.asciidoc

```diff
@@ -0,0 +1,181 @@
+[[modules-discovery-quorums]]
+=== Quorum-based decision making
+
```

Comment:
This page has some really good, key conceptual information, so I think it should be moved up before a lot of the other pages.  Maybe right after the "Bootstrapping a cluster" page.  Maybe even sooner.